cmd/config/node-density-udn/deployment-client.yml

```diff
@@ -0,0 +1,59 @@
+kind: Deployment
```

Comment:
Do we want to add some readiness network traffic between client/server pods?, (we'd need a service for that though)

Reply:
Adding such can help to get better pod ready latency KPIs

Reply:
We are tracking the data plane on https://issues.redhat.com/browse/PERFSCALE-3439
Let's add this on a followup PR if that's ok

Reply:
services are not yet fully supported by UDN iirc
can add in following PRs cc @venkataanil